etc/os-release

```diff
@@ -1,14 +1,14 @@
 NAME="elementary OS"
-VERSION="6 Odin"
+VERSION="6.1 Jólnir"
 ID=elementary
 ID_LIKE=ubuntu
-PRETTY_NAME="elementary OS 6 Odin"
+PRETTY_NAME="elementary OS 6 Jólnir"
 LOGO=distributor-logo
-VERSION_ID="6"
+VERSION_ID="6.1"
 HOME_URL="https://elementary.io/"
 DOCUMENTATION_URL="https://elementary.io/docs/learning-the-basics"
 SUPPORT_URL="https://elementary.io/support"
-BUG_REPORT_URL="https://github.com/elementary/os/issues/new"
+BUG_REPORT_URL="https://github.com/elementary/triage/issues/new"
 PRIVACY_POLICY_URL="https://elementary.io/privacy-policy"
-VERSION_CODENAME=odin
+VERSION_CODENAME=jolnir
```

Comment:
Don't want to see how everything might fall apart with a simple accent? :stuck_out_tongue:

Reply:
Lmao yeah that seems like tempting fate. I'm trained by now to not include the accent in my name even ha